spec/requests/posts_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
```

Comment:
- [ ] You currently have 4 out of 8 tests failing. Please take a look below and fix them.

![image](https://user-images.githubusercontent.com/83803414/178354785-05662be9-5868-48c7-a9c1-da197e08861d.png)

Reply:
![rspec](https://user-images.githubusercontent.com/51437483/178363619-0d81e3d3-5d37-469f-9478-5485d8d60058.JPG)
![test for controllers](https://user-images.githubusercontent.com/51437483/178363636-8f18bfab-90c5-4f0e-837c-8a8cfc29962f.JPG)

Reply:
![test for controllers](https://user-images.githubusercontent.com/51437483/178364107-8416038f-f224-42d7-bb1d-cfa3a2e44727.JPG)
![rspec](https://user-images.githubusercontent.com/51437483/178364110-498b1d8a-459d-404a-beb2-2f91f7b76d8c.JPG)

Reply:
As you see all of the tests are passed in my environment